packages/plugins/src/components/test/plugin-area.js

```diff
@@ -1,6 +1,7 @@
 /**
  * External dependencies
  */
+// eslint-disable-next-line testing-library/no-manual-cleanup
```

Comment:
We intend to use `cleanup()` in `afterEach()` here since the default way it automatically cleans up doesn't work for this scenario.